src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -131,6 +131,48 @@ def forward(self, x):
         return self.cv3(torch.cat((self.m(self.cv1(x)), self.cv2(x)), dim=1))
 
 
+class CSPLayer(nn.Module):
+    """C3 in yolov5, CSP Bottleneck with 3 convolutions"""
+
+    def __init__(
```

Comment:
- Don't like the "n" thing.
- Activation type didn't have a default, so let's not add it, add a type hint of `Type[nn.Module]` instead.
- I think we use reStructuredText docstring everywhere in SG, but whatever you like:)

Reply:
in https://github.com/Deci-AI/super-gradients/pull/248/commits/dfe766323e84dfc0085f9fe28eb50ca3a95d766e

- Renamed to num_bottlenecks.
- Removed default.
- Added docs.